Mathlib/RingTheory/Kaehler.lean

```diff
@@ -400,19 +400,19 @@ local instance smul_SSmod_SSmod : SMul (S ⊗[R] S ⧸ KaehlerDifferential.ideal
 @[nolint defLemma]
 local instance isScalarTower_S_right :
     IsScalarTower S (S ⊗[R] S ⧸ KaehlerDifferential.ideal R S ^ 2)
-      (S ⊗[R] S ⧸ KaehlerDifferential.ideal R S ^ 2) := IsScalarTower.right
+      (S ⊗[R] S ⧸ KaehlerDifferential.ideal R S ^ 2) := Ideal.Quotient.isScalarTower_right
```

Comment:
Can you explain in the PR description why these changed?

Reply:
I am not sure how to explain. I had to change it because it stopped working after the introduction of the definition of tensor product for CommSemirings.

Reply:
Still can't delete these?

Reply:
I think there's a typeclass diamond appearing here;
```
  convert IsScalarTower.right
  sorry
```
suceeds and leaves some weird goals to close.

Reply:
That's what I thought. But a `by exact?` Succeeded on finding `Ideal.Quotient.isScalarTower_right`

Reply:
Hmm, actually it's an inference issue elsewhere:
```lean
  convert IsScalarTower.right
  rotate_left 3
  · infer_instance
  · infer_instance -- fails, but `exact Ring.toSemiring` works
  · infer_instance
  · rfl
  · rfl
  · rfl
`````